BREAKING-CHANGES.md

```diff
@@ -1,8 +1,8 @@
 # Breaking Changes
 
 This document contains a list of breaking changes in each version and some notes to help migrate
-between versions. It is compiled manually from the commit history and changelog. We also tag PRs on
-github with a [breaking change] label.
+between versions. It is compiled manually from the commit history and change log. We also tag PRs on
```

Comment:
We use `changelog` in some other places. I would prefer we revert this change to keep things consistent.

Reply:
```suggestion
between versions. It is compiled manually from the commit history and changelog. We also tag PRs on
```